dcr/scenarios/agent-bvt/test_agent_basics.py

```diff
@@ -48,7 +48,7 @@ def check_root_login():
     print(root_passwd_line)
     root_passwd = root_passwd_line.split(":")[1]
 
-    if "!" in root_passwd or "*" in root_passwd:
+    if any(val in root_passwd for val in ("!", "*", "x")):
```

Comment:
Mariner 1.0 uses a different syntax to mean the same thing; `x` is not the result of a hash and therefore root password is disabled. Maybe we want to have a less explicit check here, so that we don't need to keep updating this code? Something to the effect of checking the length of the entry _might_ work (i.e. anything a single character long is not a hash), but I'm not sure. Thoughts?

Reply:
why do we need this test at all? what is its equivalent in DCR v1?

Reply:
The equivalent is: https://msazure.visualstudio.com/DefaultCollection/One/_git/Compute-Runtime-Tux-Pipeline?path=%2Fdungeon_crawler%2Fscenarios%2Fagent-bvt%2F05_check_root_login.py&_a=contents&version=GBdevelop

Reply:
My first impression is that we can drop this test. Before doing so, though, could you check if the agent disables root login (maybe during provisioning)?

Reply:
Sure, will do